include/lcp-catlistdisplayer.php

```diff
@@ -261,7 +261,7 @@ public function get_post_title($single, $tag = null, $css_class = null,
     }
 
     // Shortcode parameters take precedence.
-    $tag = esc_attr($this->params['title_tag']) ?: esc_attr($tag);
+    $tag = tag_escape($this->params['title_tag']) ?: tag_escape($tag);
```

Comment:
I looked at the source code of both `tag_escape` and `esc_attr`. The former is more strict but I think `esc_attr` already prevents XSS. I would have used `tag_escape` but I didn't know it existed because I used [WordPress Developer Resources](https://developer.wordpress.org/apis/security/escaping/) where this function is not mentioned...